src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -48,10 +49,20 @@
     <DebugEngines>{2E36F1D4-B23C-435D-AB41-18E608940038}</DebugEngines>
   </PropertyGroup>
 
+  <ItemGroup>
+    <TestTargetFrameworkPath Include="$(TestPath)%(TestTargetFramework.Folder)"/>
```

Comment:
We haven't fully defined any others yet but I would expect there will be multiple TestTargetFramework items each with their own folder.